src/commands/slow-mode.ts

```diff
@@ -0,0 +1,599 @@
+	});
+
+	if ((!slowMode || !slowMode.enabled) && channel.rateLimitPerUser === 0) {
+		throw new Error(`Slow mode for <#${channel.id}> is already disabled`);
+	}
+
+	if (slowMode && slowMode.enabled === true) {
+		slowMode.enabled = false;
+		await slowMode.save();
+	}
+
+	await channel.setRateLimitPerUser(0);
+
+	const auditLogEmbed = new EmbedBuilder()
+		.setColor(0xC0C0C0)
+		.setTitle('Event Type: _Slow-Mode Disabled_')
+		.setDescription('――――――――――――――――――――――――――――――――――')
+		.setFields(
+			{
+				name: 'User',
+				value: `<@${interaction.user.id}>`
+			},
+			{
+				name: 'Channel',
+				value: `<#${channel.id}>`
+			}
+		).setFooter({
+			text: 'Pretendo Network',
+			iconURL: interaction.guild!.iconURL()!
+		});
+
+	await sendEventLogMessage(interaction.guild!, channel.id, auditLogEmbed);
+
+	await interaction.followUp({ content: `Slow mode disabled for <#${channel.id}>` });
+}
+
+async function slowModeStatsHandler(interaction: ChatInputCommandInteraction): Promise<void> {
+	await interaction.deferReply({ ephemeral: true });
+
+	const channel = interaction.options.getChannel('channel', false, [ChannelType.GuildText]) ?? interaction.channel;
+	if (!channel) {
+		throw new Error('No channel given');
+	}
+
+	if (channel.type !== ChannelType.GuildText) {
+		throw new Error('Slow mode only applies to text channels');
+	}
+
+	const slowMode = await SlowMode.findOne({
+		where: {
+			channel_id: channel.id
+		},
+		include: 'stages'
+	});
+
+	if (channel.rateLimitPerUser === 0) {
+		if (!slowMode || !slowMode.enabled) {
+			throw new Error(`No slow mode set for <#${channel.id}>`);
+		}
+	}
+
+	const embed = new EmbedBuilder()
+		.setTitle('Slow mode stats')
+		.setFields([
+			{
+				name: 'Channel',
+				value: `<#${channel.id}>`
+			},
+		])
+		.setFooter({
+			text: 'Pretendo Network',
+			iconURL: interaction.guild!.iconURL()!
+		});
+	
+	if (slowMode) {
+		if (slowMode.enabled) {
+			if (slowMode.users) {
+				embed.addFields([
+					{
+						name: 'Participating Users',
+						value: slowMode.users.toString()
+					}
+				]);
+			}
+			
+			if (slowMode.rate) {
+				embed.addFields([
+					{
+						name: 'Current Rate',
+						value: slowMode.rate.toString()
+					}
+				]);
+			}
+		}
+
+		embed.addFields([
+			{
+				name: 'Current Limit',
+				value: `1 message every ${channel.rateLimitPerUser} seconds`
+			},
+			{
+				name: 'Window',
+				value: `${(slowMode.window / 1000).toString()} seconds`
+			}
+		]);
+	} else {
+		embed.addFields([
+			{
+				name: 'Current limit',
+				value: `1 message every ${channel.rateLimitPerUser} seconds`
+			}
+		]);
+	}
+
+	if (slowMode && slowMode.stages && slowMode.stages.length > 0) {
+		const stagesMessage = slowMode.stages.sort((a, b) => a.threshold - b.threshold).map(stage => {
+			return `1 message every ${stage.limit}s above ${stage.threshold} messages per minute`;
+		}).join('\n');
+	
+		embed.addFields([
+			{
+				name: 'Configured stages',
+				value: stagesMessage
+			}
+		]);
+	}
```

Comment:
I think this embed could be more useful if it always showed the configured stages, no matter which mode is currently set.

It could also explicitly show the currently set mode (disabled, static, or auto) as the first field.